app/Http/Controllers/Equipment/RepairsController.php

```diff
@@ -153,4 +168,26 @@ private function updateBreakageStatus(Breakage $breakage, Request $request)
         ]);
         Notify::success('Breakage updated');
     }
+
+    /**
+     * Stream an image.
+     * 
+     * @param $id
+     * @param $image
+     * 
+     * @return mixed
+     */
+    public function streamImage($id, $imageId) {
+        $breakage = Breakage::findOrFail($id);
+        $this->authorize('view', $breakage);
+
+        $image = $breakage->images()->where('position_id', $imageId)->firstOrFail();
```

Comment:
I'm sorta in 2 minds, but I feel like this can just be the image's PK rather than the position, but I'm open to debate :slightly_smiling_face: